src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < 3; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (int i = 0; i < 6; i++) {
+            figures[i] = figureSupplier.getDefaultFigure();
+        }
```

Comment:
This loop is overwriting the figures created in the previous loop. The second loop should start with 'i = 3' to fill the remaining positions in the array.